metanorm/normalizers/sentinel_safe.py

```diff
@@ -61,13 +61,14 @@ def get_instrument(self, raw_attributes):
         """Get the instrument from the attributes'"""
         if set(['Instrument']).issubset(raw_attributes.keys()):
             # This is ugly but pythesint can't find C-SAR from 'SAR-C'
+            #There is a similar code in "setinel1_identifier" normalizer
             if 'SAR-C' in raw_attributes['Instrument']:
                 instrument = 'C-SAR'
             else:
                 instrument = raw_attributes['Instrument']
 
             additional_keywords = []
-            if set(['Satellite name']).issubset(raw_attributes.keys()):
+            if set(['Satellite name']).issubset(raw_attributes.keys()) and raw_attributes['Satellite name']!='':
```

Comment:
If you do it the following way, it will also work in case `raw_attributes['Satellite name']` is None
```suggestion
            if set(['Satellite name']).issubset(raw_attributes.keys()) and raw_attributes['Satellite name']:
```
But is this test even necessary? Did you find a case where it is a problem?

Reply:
You are right, I omit that ending part of the line. It is not necessary.

Reply:
Done in fcca2cd